src/Bindings/RabbitMQAsyncCollector.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
+
+namespace Microsoft.Azure.WebJobs.Extensions.RabbitMQ
+{
+    public class RabbitMQAsyncCollector : IAsyncCollector<string>

```

Comment:
This needs to be generic type and not string similar to https://github.com/Azure/azure-webjobs-sdk-extensions/blob/dev/src/WebJobs.Extensions.CosmosDB/Bindings/CosmosDBAsyncCollector.cs#L14. 
Extending from IAsyncCollector<string> restricts using output binding to strings

Reply:
Agree with @brettsam . Lets add support for IAsyncCollector<byte[]>

Reply:
Does this mean we want users to convert whatever messages they're sending into bytes before using the output binding? Or do we take whatever type they pass in and convert it into byte[]

Reply:
For using AsyncCollector directly, we should only support  byte[]

Reply:
@katiecai - Have you tried 
`public class RabbitMQAsyncCollector : IAsyncCollector<byte[]>` ? This should work.